pymc3/tests/test_distributions.py

```diff
@@ -944,17 +944,39 @@ def test_lkj(self, x, eta, n, lp):
 
     @pytest.mark.parametrize('n', [2, 3])
     def test_dirichlet(self, n):
-        self.pymc3_matches_scipy(Dirichlet, Simplex(
-            n), {'a': Vector(Rplus, n)}, dirichlet_logpdf)
+        self.pymc3_matches_scipy(
+            Dirichlet,
+            Simplex(n),
+            {'a': Vector(Rplus, n)},
+            dirichlet_logpdf
+        )
+
+    @pytest.mark.parametrize('n', [3, 4])
+    def test_dirichlet_init_fail(self, n):
+        with Model():
+            with pytest.raises(ValueError) as err:
```

Comment:
You can pass a `match` kwarg in the `pytest.raises` context manager and avoid storing it as `err` and later calling the `match` method. You can look at the [pytest docs](https://docs.pytest.org/en/latest/assert.html#assertions-about-expected-exceptions)